tests/ballerina-integration-test-utils/src/main/java/org/ballerinalang/test/context/BServerInstance.java

```diff
@@ -68,7 +69,7 @@ private void initialize() throws BallerinaTestException {
     }
 
     private void configureAgentArgs() throws BallerinaTestException {
-        String balAgent = System.getProperty("ballerina.agent.path");
+        String balAgent = Paths.get(System.getProperty("ballerina.agent.path")).toString();
```

Comment:
Should we define "ballerina.agent.path" as a constant ?

Reply:
Can add it.. But this is the only place this string is used

Reply:
+1 for moving to a constant (a common place). Then we know where to find all the properties if we ever wanted to change it, even if its used only once..